frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenuCards/FeatureStrategyMenuCards.tsx

```diff
@@ -22,10 +23,12 @@ export const FeatureStrategyMenuCards = ({
     projectId,
     featureId,
     environmentId,
+    onlyReleasePlans,
```

Comment:
I think this is fine, but I was wondering, what if we just want to display strategies?
I'd think of something like:
```typescript
displayComponents = ['strategies', 'releasePlans']
```
as a default value but you can choose... Alternatively
```typescript
displayComponents = { strategies: true, releasePlans: true }
```

Reply:
I'm open to this, but only when that becomes a requirement. Let's leave this one for when we have a better sense of what we're doing with the UI here

Reply:
Alternative: This component could accept `children` so that the caller decides what will be displayed here.

Reply:
But remember that this will be rebuilt soon, so it probably doesn't matter much anyways.

Reply:
> But remember that this will be rebuilt soon, so it probably doesn't matter much anyways.

Honestly this is what's going through my mind. I'd rather not make this open to change when the next planned change is to throw this away. Right now it needs to get us over the initial feedback phase so we know what we should do next